src/styles/common/_pills.scss

```diff
@@ -66,7 +66,7 @@ $pillTextColorDark: lighten($black, 31%);
     }
 }
 
-.pill-error {
+.bdl-Pill--error {
```

Comment:
I'm not seeing any usages of this class in the codebase?

Reply:
Do you know if it's used in any other codebases?

Reply:
Not sure. It'd probably be good to do a quick global search in our main app codebases. If it's not used, we should delete it.

Reply:
We were looking at using the per-pill error state for task assignment

Reply:
Thanks for the information, @alexkrolick. I will keep this class.